packages/database/src/realtime/BrowserPollConnection.ts

```diff
@@ -386,7 +393,7 @@ export class BrowserPollConnection implements Transport {
   }
 }
 
-export interface IFrameElement extends HTMLIFrameElement {
+export interface FrameElement extends HTMLIFrameElement {
```

Comment:
Hm. A Frame is different from an IFrame. Can we keep the original name?

Reply:
lol. that's what happens after working brainlessly for 2 hrs.
Fixed.